ui/app/scripts/app/controllers/apps.js

```diff
@@ -81,7 +81,7 @@ define(['model/pageable'], function (Pageable) {
 
                                 // Define function for obtaining properties of a app
                                 item.getProperties = function() {
-                                    return appService.getAppInfo(this.type, this.name);
+                                    return appService.getAppInfo(this.type, this.name).$promise;
```

Comment:
`$http` service returns a promise already. I have switched it to `$resource` service that seems more appropriate and it returns an object with a `$promise` property.